client/components/map/WebMap.web.tsx

```diff
@@ -32,7 +32,7 @@ import {
   multiPolygonBounds,
 } from '../../utils/mapFunctions';
 import MapButtonsOverlay from './MapButtonsOverlay';
-import { saveFile } from '../../utils/fileSaver/fileSaver';
+import { saveFile } from '../../utils/fileSaver/fileSaver.native';
```

Comment:
we should not do this, react native resolves the .native and .web extensions per platform so it doesn't get compiled.

Reply:
to fix the type error we need to make these into folders with index and custom exports I believe